tests/framework/ROM/TimeSeries/ARMA/Multiyear/dataSet.xml

```diff
@@ -0,0 +1,5 @@
+<data name="windRawData" type="HistorySet">
+<input>scaling</input>
+<output>Time,Speed</output>
+<inputFilename>dataSet.csv</inputFilename>
+</data>
```

Comment:
Only thing I find, now ...

Reply:
Added a newline; I'll commit it when I change the "growth" input.